messaging-client-core/src/main/java/com/inmobi/messaging/Partition.java

```diff
@@ -0,0 +1,21 @@
+package com.inmobi.messaging;
+
+import java.io.Serializable;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Partition Identifier of a source of data.
+ *
+ */
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Getter
+public class Partition implements Serializable {
+
+  private final String partitionId;
```

Comment:
Shouldn't this be of type Integer? Any particular reason for String.

Reply:
partition id is string in Eventhub kestrel

Reply:
Generalizing the id as String.